backend/app/adapter/db/migration/12_create_changelog_table.sql

```diff
@@ -0,0 +1,11 @@
+-- +migrate Up
+CREATE TABLE "change_log"
+(
+    "id"                CHARACTER VARYING(5) PRIMARY KEY,
```

Comment:
Why do we use `VARCHAR` instead of `SERIAL` for `id`?

Reply:
We are generating Key manually using KeyGenerator

Reply:
Oh, got it. Cool:)

Reply:
This is for scalability reason.